railties/test/application/test_runner_test.rb

```diff
@@ -284,6 +354,18 @@ def test_fixture
         RUBY
       end
 
+      def create_backtrace_test
```

Comment:
the output of the tests that check backtrace look the same when backtrace is enable or disabled. Thats looks weird IMO. We should have different outputs I think.